style.css

```diff
@@ -0,0 +1,48 @@
+*,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;

```

Comment:
- [ ] it is better to add the `font-family` property to the body element and all children elements are going to inherit that property by default. ♻️ ‏